### PR TITLE
Reenable graphql query and subscription tests 

### PR DESCRIPTION
**Pull request by gianalarcon**
-Update graphql query for entities and models
-Update graphql subscription for entities and models

Solves #1021 

This PR is not finished yet. ToDo:
1. There is one test that is ignored for now
https://github.com/gianalarcon/dojo/blob/16aae95a8557b8f9fd3e23f03f7c98fff2408d09/crates/torii/graphql/src/tests/models_test.rs#L125
```
  async fn test_model_ordering(pool: SqlitePool) {
```
This test depends on https://github.com/gianalarcon/dojo/blob/16aae95a8557b8f9fd3e23f03f7c98fff2408d09/crates/torii/graphql/src/query/data.rs#L55
```
pub async fn fetch_multiple_rows()
```
I think we have to update/refactor above function because it uses column_name `external_vec` to perform ordering, but it should use `external_x ` and `external_y`
@broody What do you think about it. any suggestions?

### CODE REVIEW

**Comment by broody** [comment]
Right, a current limitation is not being able to filter/order on nested types because subqueries are used to retrieve their data. It's related to this [issue](https://github.com/dojoengine/dojo/issues/968), once it's resolved I can enable those tests again.

**Comment by gianalarcon** [comment]
Hi @broody. I enabled most of the test, I think I can deliver these tests now. After the issue that you mentioned is solved I can deliver the last test.

**Comment by broody** [comment]
> Hi @broody. I enabled most of the test, I think I can deliver these tests now. After the issue that you mentioned is solved I can deliver the last test.

Yeah agreed, thx!

File: crates/torii/graphql/src/tests/models_test.rs [hunk 1]
```diff
@@ -46,54 +51,78 @@ mod tests {
...
-            ("where: { xLT: 42 }", 0),
-            ("where: { xLTE: 42 }", 1),
-            ("where: { x: 1337, yGTE: 1234 }", 0),
-            (r#"where: { player: "0x2" }"#, 1), // player is a key
+            (
+                r#"where: { playerNEQ: "0x0000000000000000000000000000000000000000000000000000000000000002" }"#,
+                1,
+            ),
+            (
+                r#"where: { playerGT: "0x0000000000000000000000000000000000000000000000000000000000000002" }"#,
```
Comment (broody) [inline crates/torii/graphql/src/tests/models_test.rs:82]: Ah we should trim zeros on input objects too, much like graphql returned results. We can do that in another PR

**Review by broody — APPROVED** [review]
